labs/computing-experiment-subjects/README.md

```diff
@@ -0,0 +1,76 @@
+# Computing Experiment Datasets #1: Experiment Subjects
+
+This Lab is part of a multi-part series focused on computing useful experiment datasets. In this Lab, we'll use [PySpark](https://spark.apache.org/docs/latest/api/python/index.html) to compute _experiment subjects_ from [Optimizely Enriched Event](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) ["Decision"](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification#decisions-2) data.
```

Comment:
Is experiment subjects the term we want to centralize on as a part of our documentation? What happens in a world where someone is getting analytics on feature flags being on / off, should they still be `experiment subjects`? Maybe worth thinking about.

Reply:
That is a good point, but for now I'd rather use experiment subjects, since e3 data is not even available for feature flags outside of experiments.  Within the context of experiment analysis, the term is the right one.  If, later on, we decide to use a different umbrella term we can always update this and other labs accordingly.

cc @loganlinn